v1/pdf.yaml

```diff
@@ -61,7 +61,7 @@ paths:
             request:
               method: get
               # Note: The title needs to be encoded twice.
-              uri: '{{options.uri}}/pdf?accessKey={options.secret}&url=https://{{domain}}/wiki/{_encodeURIComponent(title)}%3Fprintable=yes'
+              uri: '{{options.uri}}/pdf?accessKey={options.secret}&url={{options.scheme}}://{{domain}}/wiki/{_encodeURIComponent(title)}%3Fprintable=yes'
```

Comment:
Could you update it to default to https? something like `{{default(options.scheme, "https")}}`?